### DISCUSSION TITLE
index

### DISCUSSION THREAD

**Discussion by giscus** [kananinirav/AWS-Certified-Cloud-Practitioner-Notes#239] [Comments]
# index

If you are preparing for AWS Cloud Practitioner Exam, this guide will help you with quick revision before the exam. I share the notes I used to study and pass exam.

https://kananinirav.com/

**Comment by FSkateP** [comment]
Hi

I attempted my AWS last year and failed by 2%. Is there any free
certificates i can try?

**Comment by chojuninengu** [comment]
Alright, thanks alot, i will use this free resource too

**Comment by lele-maxwell** [comment]
looks good, thanks
